sections/01_abstract.md

```diff
@@ -1,3 +1,18 @@
 ## Abstract
 
-Abstract goes here.
+Deep learning, a new class of machine learning algorithms, has recently showed
+impressive results across a variety domains. Biomedicine and genomics, both
+data- and feature-rich and yet complex and ill-understood, presents an obvious
+target for applying this new approach. We examine applications of deep learning
+to a variety of biomedical data problems - in patient classification,
+fundamental research and patient treatment -  to determine if similar progress
+can be made here or what unique challenges the biomedical sphere holds. We find
+that while deep learning has achieved (sometimes remarkable) advances on prior
+state-of-the-art methods, it has not yet revolutionized or definitively solved
+any problem. In some cases, this is probably simply a matter of time but
+elsewhere technical advances need to be made in the areas of interpretability,
+unsupervised learning and most generally how to best model a problem. Beyond the
+technical, the limited amount of (correctly labeled) data available for traing
```

Comment:
Missing word (problems/issues?) after `technical`? 

traing --> training

I would also suggest:
> labelled

instead of
>(correctly labelled)